### PR TITLE
Register Argo CD topic and other updates

### PR DESCRIPTION
**Pull request by MelwynPaul**
Added Register Argo CD topic
Added Install Multicluster Verrazzano section and moved MC topics
Fixed all relevant links to MC topics

### CODE REVIEW

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        enabled: true
+EOF
+```
+</div>
+{{< /clipboard >}}
+
+
+## Refresh the Rancher API token
+
+In the cluster operator's helm chart, you can update the configurable `ARGOCD_CLUSTER_TOKEN_TTL` token's `argoCDClusterTokenTTL` value. By default, the TTL value is set to 240 minutes. In the controller, override the Argo CD token's TTL to 600 minutes.
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/mc-install/register-argocd.md:43]: If this is something that users **will** want to do, this section needs to be re-written from the user's POV. 
First, explain what happens when the Argo CD token's TTL expires and why would/should users change it?
Then -> By default, the Argo CD token's TTL is set to 240 minutes. If you want to modify it, you can update the configurable `ARGOCD_CLUSTER_TOKEN_TTL` token's `argoCDClusterTokenTTL` value. The following example show you how to override the Argo CD token's TTL to 600 minutes.
Last, is there a reason why this section needs to be called Refresh the Rancher API token? 
Can it be something like, How to override the Argo CD TTL

**Review by rosemarymarano — COMMENTED** [review]
Some comments/edits. There are some areas where you should obtain the requested information from Preethi or Lelia.

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Register Argo CD in a Multicluster Verrazzano Environment"
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: Yes, I agree. Updated. Thank you.

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+## Prerequisites
+
+- Make sure you have completed the steps in the Prerequisites and Install Verrazzano sections in [Install Multicluster Verrazzano]({{< relref "/docs/setup/install/mc-install/multicluster.md" >}}).
+- [Enable Argo CD](#enable-argo-cd).
+
+### Enable Argo CD
+
+To use Argo CD in a multicluster Verrazzano environment, you must first enable it on the admin cluster.
+
+Argo CD is _not_ enabled by default, use the following example to enable it using the `dev` installation profile.
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: Updated. Thank you.

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        - values:
+            argoCDClusterTokenTTL: 600
+```
+
+</div>
+{{< /clipboard >}}
+
+## Deregister a cluster
+
+To deregister a cluster in Argo CD, deregister the managed cluster. See [Deregister a managed cluster]({{< relref "/docs/setup/install/mc-install/multicluster#deregister-a-managed-cluster" >}}).
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: Oh. My bad. 
Updated. Thank you.

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+EOF
+```
+</div>
+{{< /clipboard >}}
+
+
+## Override the Argo CD TTL
+
+By default, the Argo CD token's TTL is set to 240 minutes. If you want to modify it, you can update the configurable `ARGOCD_CLUSTER_TOKEN_TTL` token's `argoCDClusterTokenTTL` value. The following example show you how to override the Argo CD token's TTL to 600 minutes.
+
```
Comment (MelwynPaul) [inline content/en/docs/setup/install/mc-install/register-argocd.md:44]: @preethignanesh @leliyin 
We need your help to update this section. Please provide information. 

Following is a comment from @rosemarymarano.
If this is something that users **will** want to do?
Explain what happens when the Argo CD token's TTL expires and why would/should users change it?

File: content/en/docs/setup/install/mc-install/multicluster-no-rancher.md [hunk 6]
```diff
@@ -282,12 +282,12 @@ managed cluster, with the following information:
 
 ### Verify that managed cluster registration has completed
 
-After these steps have been completed, return to [Verify that managed cluster registration has completed]({{< relref "/docs/setup/install/multicluster.md#verify-that-managed-cluster-registration-has-completed" >}}).
+After these steps have been completed, return to [Verify that managed cluster registration has completed]({{< relref "/docs/setup/install/mc-install/multicluster.md#verify-that-managed-cluster-registration-has-completed" >}}).
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/multicluster-no-rancher.md:286]: Thank you, @leliyin.
We plan to add this new section via another update/PR.

File: content/en/docs/setup/install/mc-install/multicluster-no-rancher.md [hunk 7]
```diff
@@ -282,12 +282,12 @@ managed cluster, with the following information:
...
 
 ## Deregister a managed cluster without Rancher
 
 **NOTE**: The following procedure is for a Verrazzano multicluster environment in which Rancher is not enabled on the admin cluster.
-If Rancher is enabled, for a much simpler deregistration process, see [Deregister a managed cluster]({{< relref "docs/setup/install/multicluster.md#deregister-a-managed-cluster" >}}).
+If Rancher is enabled, for a much simpler deregistration process, see [Deregister a managed cluster]({{< relref "docs/setup/install/mc-install/multicluster.md#deregister-a-managed-cluster" >}}).
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/multicluster-no-rancher.md:290]: Thank you, @leliyin.
We plan to add this new section via another update/PR.

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+        - values:
+            argoCDClusterTokenTTL: 600
+```
+
+</div>
+{{< /clipboard >}}
+
+## Deregister a cluster
+
+To deregister a cluster in Argo CD, follow the instructions at [Deregister a managed cluster]({{< relref "/docs/setup/install/mc-install/multicluster#deregister-a-managed-cluster" >}}).
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md:44]: We will add this info (link to it) after we have the relevant section added in the multicluster-no-rancher topic.

File: content/en/docs/setup/install/mc-install/register-argocd.md (hunk 5 above)
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: Thank you, @leliyin.
As this section needs more work, the plan is to remove the 'Override the Argo CD TTL' and update it later via another update/PR. 
We need to finalize and publish the Enable Argo CD info in this 'Register Argo CD' topic.

File: content/en/docs/setup/install/mc-install/register-argocd.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (desagar) [reply content/en/docs/setup/install/mc-install/register-argocd.md:44]: I thought Rancher not being enabled is not even a case we need to consider. At the top of hte document we state that ArgoCD depends on Rancher being enabled. Or perhaps I'm misunderstanding this.

Also, registering a managed cluster during multicluster set up (as well as deregistering) both automatically get done when you do the multicluster registration. We should make clear that the user doesn't need to go do this registration or deregistration again just for ArgoCD .

File: content/en/docs/setup/install/mc-install/register-argocd.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Register Argo CD"
```
Comment (desagar) [inline content/en/docs/setup/install/mc-install/register-argocd.md:2]: I'm not sure what "Register Argo CD" means exactly. We don't register Argo CD. We do the following:
1. Enable ArgoCD in Verrazzano and
2. Register managed clusters with Argo CD once it is enabled.

Both of these are prerequisites to being able to use Argo CD multicluster application deployment

Comment (rosemarymarano) [reply content/en/docs/setup/install/mc-install/register-argocd.md:2]: Agree. A good and important point.

File: content/en/docs/setup/install/mc-install/register-argocd.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (leliyin) [reply content/en/docs/setup/install/mc-install/register-argocd.md:44]: Agreed. We don't need a section to deregister a cluster in Argo CD.  This is done automatically when the user deletes the vmc resource. 

For the 'rancher-not-enabled' on the admin cluster section, I think we need to add a section to manually register and de-register a managed cluster in argocd.

File: content/en/docs/setup/install/mc-install/register-argocd.md (hunk 9 above)
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: @desagar @rosemarymarano 
Thank you. 
Yes, I agree too. 
The title has come a full circle. We initially had the title as "Enable Argo CD" 👍

File: content/en/docs/setup/install/mc-install/register-argocd.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/mc-install/register-argocd.md]: @desagar 
Yes, we thought the same - "Rancher not being enabled is not even a case".
From @leliyin's 'add sections' request, looks like we have them.

We can discuss and add them - after we push out this 'Enable Argo CD' info, which users are looking for.

**Comment by MelwynPaul** [comment]
@desagar @leliyin @preethignanesh @kminder 

After careful analysis, we feel we can do away with all these changes -  by updating the Argo CD sample topic  (Examples > Argo CD) topic > to point to general Pre-Installation info, which is in the 'Modify Verrazzano Installations' topic. 

See PR https://github.com/verrazzano/docs/pull/880

**Pull request closed without merging**